### ISSUE TITLE
traci.lane feature request

### ISSUE THREAD

**Issue by cspatharis** [flow-project/flow#785]
Hello,

I'm trying to get the "getLastStepVehicleIDs()" function but I can't find it in FLOW. In SUMO's traci I know it is located in _lane python file. Can I integrate it in any way in my FLOW repo?

Thanks in advance.

**Comment by eugenevinitsky** [comment]
Yep! It's a small change actually, but would require a tiny of work. If you look in the TraciVehicle class you can see in the _add_departed method all of the SUMO objects we subscribe to. If you add a subscription to your desired sumo constant, you will then be able to extract and save it in the update method of that class. Then, you just define a getter for this item.

**Comment by eugenevinitsky** [comment]
Since this is a feature that might require a bit of discussion, I might direct you to the Flow-users group for faster discussion: https://join.slack.com/t/flow-users/shared_invite/enQtNzk5NzE1OTkyNTE0LTU0YmE5ZTVkZmJmNDg1Nzg2YThjOThmY2ZlYWE2ZjQ0NTE1NjE3Y2YwMTljY2U5M2Y3MDhmMWUzZDM3YWViNmE
